nodes/pods/nodes-pods-autoscaling-custom.adoc

```diff
@@ -8,24 +8,34 @@ toc::[]
 
 As a developer, you can use the custom metrics autoscaler to specify how {product-title} should automatically increase or decrease the number of pods for a deployment, stateful set, custom resource, or job based on custom metrics that are not based only on CPU or memory.
 
-:FeatureName: Scaling by using a scaled job
-include::snippets/technology-preview.adoc[]
-
 The Custom Metrics Autoscaler Operator for Red Hat OpenShift is an optional operator, based on the Kubernetes Event Driven Autoscaler (KEDA), that allows workloads to be scaled using additional metrics sources other than pod metrics.
 
+[NOTE]
+====
 The custom metrics autoscaler currently supports only the Prometheus, CPU, memory, and Apache Kafka metrics.
+====
 
 // For example, you can scale a database application based on the number of tables in the database, scale another application based on the number of messages in a Kafka topic, or scale based on incoming HTTP requests collected by {product-title} monitoring.
 
-:FeatureName: Autoscaling based on Apache Kafka metrics
+:FeatureName: The custom metrics autoscaler
 include::snippets/technology-preview.adoc[leveloffset=+0]
 
 // The following include statements pull in the module files that comprise
 // the assembly. Include any combination of concept, procedure, or reference
 // modules required to cover the user story. You can also include other
 // assemblies.
 
-include::modules/nodes-pods-autoscaling-custom-rn.adoc[leveloffset=+1]
+include::modules/nodes-pods-autoscaling-custom-rn-210.adoc[leveloffset=+1]
+
+.Additional resources
+
+* xref:../../nodes/pods/nodes-pods-autoscaling-custom.adoc#nodes-pods-autoscaling-custom-metrics-access_nodes-pods-autoscaling-custom[Accessing performance metrics]
+* xref:../../nodes/pods/nodes-pods-autoscaling-custom.adoc#nodes-pods-autoscaling-custom-pausing_nodes-pods-autoscaling-custom[Pausing the custom metrics autoscaler for a workload]
+* For information on using fallback and HPA naming, see xref:../../nodes/pods/nodes-pods-autoscaling-custom.adoc#nodes-pods-autoscaling-custom-creating-workload_nodes-pods-autoscaling-custom[Adding a custom metrics autoscaler to a workload].
+* For information on activation and scaling thresholds, see xref:../../nodes/pods/nodes-pods-autoscaling-custom.adoc#nodes-pods-autoscaling-custom-about_nodes-pods-autoscaling-custom[Understanding the custom metrics autoscaler].
+
```

Comment:
**PEER REVIEW: I know this is an unusual format, I wanted to include the "For information" to be clear on why we are providing those links as additional resources. It seemed confusing to link to previously existing sections otherwise.**